README-PLAYBOOK.md

```diff
@@ -67,112 +67,127 @@ Several playbook samples are provided in the [examples folder](./examples/playbo
-| p12Task      | string | *OPTIONAL* - Used when [Connection.platform](#connection) is `tpp` to reference a configured [CertificateTask.name](#certificatetask) to be used for certificate authentication. Will be used to get a new accessToken when `accessToken` is missing, invalid, or expired. Referenced certificateTask must have an installation of type `pkcs12`. |
-| refreshToken | string     | *OPTIONAL* - Used when [Connection.platform](#connection) is `tpp` to refresh the `accessToken` if it is missing, invalid, or expired. If omitted, the `accessToken` will not be refreshed when it expires. When a refresh token is used, a new accessToken *and* refreshToken are issued. The previous refreshToken is then invalid (one-time use only). vCert will attempt to update the refreshToken and accessToken fields upon refresh. |
-| scope        | string     | *OPTIONAL* - Used when [Connection.platform](#connection) is `tpp` to determine the scope of the access token when refreshing the access token, or when getting a new grant using a `pkcs12` certificate. Defaults to `certificate:manage` if omitted.|
+### Connection
 
+| Field       | Type                               | Description                                                                                                                                                                                                                                                                                                                                                                                      |
+|-------------|------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| credentials | [Credentials](#credentials) object | ***REQUIRED*** - A [Credential](#credentials) object that defines the credentials used to connect to the selected provider `platform`.                                                                                                                                                                                                                                                           |
+| trustBundle | string                             | *OPTIONAL* - Used when [Connection.platform](#connection) is `tpp`, or `firefly`.<br/>Defines path to PEM-formatted trust bundle that contains the root (and optionally intermediate certificates) to use to trust the TLS connection. If omitted, will attempt to use operating system trusted CAs.<br/>*IGNORED* otherwise.                                                                    |
```

Comment:
```suggestion
| trustBundle | string                             | *OPTIONAL* - Used when [Connection.platform](#connection) is `tpp` or `firefly`.<br/>Defines path to PEM-formatted trust bundle that contains the root (and optionally intermediate certificates) to use to trust the TLS connection. If omitted, will attempt to use operating system trusted CAs.<br/>*IGNORED* otherwise.                                                                    |
```